content/blog/new-powershell-library-for-the-hpe-greenlake-cloud-platform.md

```diff
@@ -0,0 +1,358 @@
+  >
+  > **Note**: To store the API credentials beyond the duration of the session, the cmdlet provides a **\-Location** parameter. This parameter can be used with **New-HPEGLAPIcredential** to save the API credentials in a directory and the **\-Encrypt** parameter can be utilized to encrypt the API credentials before exporting the JSON file into the designated *Location* directory.
+
+  * **Remove-HPEGLAPIcredential**: to delete an API credential of an application instance.
+
+  > **Note**: Once API credentials are deleted, access to the application instance API is lost permanently. This is because API credentials are used to authenticate and authorize access to the API. When the credentials are deleted, the corresponding API client ID is also invalidated, which means that any requests made using those credentials will be rejected by the API server.
+
+## Managing subscriptions with the HPE GreenLake PowerShell Library
+
+The main cmdlets for managing subscriptions in the HPE GreenLake Platform are the followings:
+
+* To get information about subscriptions: 
+
+  * **Get-HPEGLDeviceSubscription**: to get information about your device subscriptions available in your HPE GreenLake account. Several parameters can be used to display the subscriptions with available quantity, the subscriptions that are expired or not expired, etc. You can combine parameters to obtain for example only the subscription keys that have not expired and for which there are still licenses available: **Get-HPEGLDeviceSubscription -Notexpired -Available**
+* To manage device subscriptions:
+
+  * **Add-HPEGLDeviceSubscription**: to add device subscription to the HPE GreenLake account.
+  * **Set-HPEGLDeviceAutoSubscription**: to set automatic subscription assignment. This feature automatically assigns a subscription to any supported device that is added to the HPE GreenLake platform.
+  * **Remove-HPEGLDeviceAutoSubscription**: to remove an automatic assignment of subscriptions.
+
+* To apply subscription key to devices:
+
+  * **Set-HPEGLDeviceSubscription**: to apply a subscription key to one or more devices. 
+
+  >  **Note**: When the auto device subscription is not is not supported or not enabled for the type of device you use, you need to manually apply a subscription key.
+
+  * **Remove-HPEGLDeviceSubscription**: to detach devices from a subscription key. 
+
+# Summary
+
+In this blog, we have introduced you to the primary cmdlets that are provided by this library. However, there are more cmdlets available and much more to explore. Nevertheless, you now possess the necessary knowledge to begin testing, to develop your own PowerShell scripts to start automating your tasks and experience greater productivity and afficiency.
```

Comment:
In the first sentence, instead of saying "by this library", I'd recommend spelling out "the PowerShell library" or "this new PowerShell library".

The last sentence needs a little work. Try rephrasing to:
"Nevertheless, you now possess the necessary knowledge to begin testing and to develop your own PowerShell scripts to start automating tasks so you can experience greater productivity and efficiency. " [note spelling of "efficiency"]